### DISCUSSION TITLE
README

### DISCUSSION THREAD

**Discussion by ion-oset** [ion-oset/developer-commons#2] [Announcements]
## Welcome to the Developer Commons discussion forum

We’re using this forum to manage and coordinate development between various [TrustTheVote® Project](https://github.com/TrustTheVote-Project) repositories, as we refine and improve our development process.

Goals of this repo include:

  - Improving and building consensus on engineering and developement best practices.
  - Tracking proposals from inception to completion, from idea to formal adoption.
  - Sharing resources, references and tools.
  - Optimizing our workflow for open source development teams, with language and GitHub specific considerations.
  - Connecting collaborators with each other.

Please take a look at the [Wiki](https://github.com/ion-oset/developer-commons/wiki) and in particular at the [proposed workflow for this forum](https://github.com/ion-oset/developer-commons/wiki#workflow).

**Comment by stratofax** [comment]
Updated with minor edits and a new link to [The TrustTheVote® Project](https://github.com/TrustTheVote-Project).
